packages/core/data-transfer/src/strapi/remote/routes.ts

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line node/no-extraneous-import
+import type { Context } from 'koa';
```

Comment:
So this imports the type from 'koa' which actually imports from '@types/koa' which is why we've removed 'koa' from package.json? It feels odd, especially since we have to eslint-disable it. Could we add it back as a devDependency and remove this eslint directive?

Reply:
I've added Koa back :tada: